### ISSUE TITLE
Multiple node selection using visNetworkProxy and visSelectNodes

### ISSUE THREAD

**Issue by martycheung** [datastorm-open/visNetwork#87]
Hi bthieurmel,

Using visNetworkProxy + visSelectNodes and the selectInput search box, it is not possible to select multiple nodes or edges. If I select multiple node id's, only the 1st selected node is highlighted. This is also the case in your demo application.

Is multiple node selection using visNetworkProxy + visSelectNodes and selectInput going to be included in a future release? Or am I somehow missing the correct way to do this?

Thanks!

**Comment by bthieurmel** [comment]
Hi.

Multiple node selection is normally available. This is a simple example : 

````
require(visNetwork)
require(shiny)

server <- function(input, output) {
  datass <- reactive({
    set.seed(2)
    nodes <- data.frame(id = 1:15, label = paste("Label", 1:15),
                        group = sample(LETTERS[1:3], 15, replace = TRUE))
    
    edges <- data.frame(id = 1:15, from = trunc(runif(15)*(15-1))+1,
                        to = trunc(runif(15)*(15-1))+1)
    list(nodes = nodes, edges = edges)
  })
  
  output$network_proxy_select <- renderVisNetwork({
    visNetwork(datass()$nodes, datass()$edges) %>% visLegend()
  })
  
  observe({
    nodes_selection <- input$selnodes
    visNetworkProxy("network_proxy_select") %>%
      visSelectNodes(id = nodes_selection)
  })
}

ui <- fluidPage(
  sidebarLayout(
    sidebarPanel(
      selectInput(inputId = "selnodes", label = "Nodes selection", choices = 1:15, multiple = TRUE)
    ),
    mainPanel(visNetworkOutput("network_proxy_select"))
  )
)

shinyApp(ui = ui, server = server)
````

But in some case, with ``highlightNearest`` options for example, you can't view this multiple selection. I fix the demo application.

**Comment by complexly** [comment]
Is it possible to mark the unselected nodes grey, just like the current highlight style, but with multiple selected nodes highlighted? 

I hacked this function awkwardly by manually update nodes data each time, but I am wondering if there is a smart way to do this. Thanks!

**Comment by bthieurmel** [comment]
Actually not. If you hacked the code, you can shared it with me to hope a integration...!

**Comment by endreanderssen** [comment]
Hi

Pardon me for butting in, but I have been struggling to select nodes correctly based on user input when I found this conversation. Running the code in the example above I found that if I choose a value larger than 10 it actually highlights two nodes in the network. If I select node 13 in the dropdown I see node 1 and 3 highlighted, if I select 14 in the dropdown it highlights node 1 and 4 and so on. 

Is anyone else seeing this behavior? 

![image](https://cloud.githubusercontent.com/assets/22217723/18551407/8de1e102-7b58-11e6-9c68-f97ce86d1aa6.png)
